elasticsearch/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
     yum clean all
 # initial boring elasticsearch.yml is just to enable plugin installation.
 # put the real config in place in a deployment with a ConfigMap.
-ADD elasticsearch.yml logging.yml /usr/share/elasticsearch/config/
+ADD elasticsearch.yml logging.yml com.redhat.viaq.template.json /usr/share/elasticsearch/config/
```

Comment:
I thought this is overwritten by the configmap mounted by the pod. Does it make sense to still add it here?